src/index.ts

```diff
@@ -11,6 +11,10 @@ import { IconH1, IconH2, IconH3, IconH4, IconH5, IconH6, IconHeading } from '@co
  * @property {string} text — Header's content
  * @property {number} level - Header's level from 1 to 6
  */
+interface HeaderData {
+  text: string;
+  level: number;
```

Comment:
please add jsdocs to each property. You can move them from the `@typedef` section which is not needed anymore